config/locales/devise.en.yml

```diff
@@ -24,43 +24,43 @@ en:
-      signed_up_but_unconfirmed: "A message with a confirmation link has been sent to your email address. Please follow the link to activate your account."
-      update_needs_confirmation: "You updated your account successfully, but we need to verify your new email address. Please check your email and follow the confirmation link to confirm your new email address."
-      updated: "Your account has been updated successfully."
-      updated_but_not_signed_in: "Your account has been updated successfully, but since your password was changed, you need to sign in again."
+      destroyed: "Your account has been cancelled. We hope to see you again soon"
+      signed_up: "You have successfully signed up for an account"
+      signed_up_but_inactive: "You have successfully signed up for an account. You won't be able to sign in until your account is activated"
+      signed_up_but_locked: "You have successfully signed up for an account. We could not sign you in because your account is locked"
+      signed_up_but_unconfirmed: "We've sent a confirmation link to your email address. Follow the link to activate your account"
+      update_needs_confirmation: "You successfully updated your account. We need to verify your new email address. Check your email and follow the link to confirm your new email address"
+      updated: "Your account has been successfully updated"
+      updated_but_not_signed_in: "Your account has been successfully updated. You'll need to sign in again"
     sessions:
       signed_in: ""
       signed_out: ""
       already_signed_out: ""
     unlocks:
-      send_instructions: "You will receive an email with instructions for how to unlock your account in a few minutes."
-      send_paranoid_instructions: "If your account exists, you will receive an email with instructions for how to unlock it in a few minutes."
-      unlocked: "Your account has been unlocked successfully. Please sign in to continue."
+      send_instructions: "You will receive an email in a few minutes with instructions for how to unlock your account"
+      send_paranoid_instructions: "If your account exists, you will receive an email in a few minutes with instructions for how to unlock it"
+      unlocked: "Your account has been successfully unlocked. Sign in to continue"
   errors:
     messages:
-      already_confirmed: "was already confirmed, please try signing in"
-      confirmation_period_expired: "needs to be confirmed within %{period}, please request a new one"
-      expired: "has expired, please request a new one"
-      not_found: "not found"
-      not_locked: "was not locked"
+      already_confirmed: "Your account has already been confirmed. Sign in"
+      confirmation_period_expired: "Your account needs to be confirmed within %{period}. Request a new account"
+      expired: "Your account has has expired. Request a new account"
```

Comment:
@Dan13L3 this actually refers to reset password token not account, and doesn't need to be added into the message it's injected in the code. 

Same goes for the others in this section that do refer to account.

Reply:
@baarkerlounger Ahh, apologies - is it simple enough to revert?

Reply:
Nothing merged yet so no need to revert anything. I think easiest is I'll try to put your intended changes into a new PR and you can double check?

Reply:
@baarkerlounger sounds good - thanks